DataFormats/Detectors/TOF/src/CalibTimeSlewingParamTOF.cxx

```diff
@@ -20,53 +20,52 @@ using namespace o2::dataformats;
 //ClassImp(o2::dataformats::CalibTimeSlewingParamTOF);
 
 CalibTimeSlewingParamTOF::CalibTimeSlewingParamTOF()
+{
+  for (int i = 0; i < NSECTORS; i++) {
+    memset(mChannelStart[i], -1, sizeof(mChannelStart[i]));
+    memset(mFractionUnderPeak[i], -100, sizeof(mFractionUnderPeak[i]));
```

Comment:
Hi @chiarazampolli, sorry, I've just noticed that the ``mSigmaPeak`` and ``mFractionUnderPeak`` are floats, for them the memset -1 will not work (and -100 will not work even for int, I have to change my glasses). 
I can fix it myself but to be sure, why do you need for them -1 and -100 to signal anomaly? Would not 0 work? 
Per-element initialization of such a large arrays is really quite time consuming.

Reply:
See https://github.com/chiarazampolli/AliceO2/pull/19